docs/metrics.md

```diff
@@ -0,0 +1,133 @@
+
+## Analysis
+
+Data collection is done solely for the purpose of product development, improvement and maintenance.
+
+More here TBD
+
+## Collection
+
+Data will be collected using this library:
+
+https://github.com/mozilla-mobile/telemetry-ios/
+
+We plan to submit two ping types.
+
+First is the "core ping", which contains information about the iOS version, architecture, etc of the device lockbox has been installed on:
+
+https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/core-ping.html
+
+The second is the "focus event ping" which allows us to record event telemetry:
+
+https://github.com/mozilla-mobile/focus-ios/wiki/Event-Tracking-with-Mozilla%27s-Telemetry-Service
+
+The ping types are defined in `lockbox-ios/Common/AppDelegate.swift`. Scheduling of ping transmission is done in the same file.
+
+Every event must contain `category`, `method` and `object` fields, and may optionally contain `value` and `extra` fields as well. Possible values for the former three fields are defined in `lockbox-ios/TelemetryIntegration.swift`
+
+For all events, the `extra` field should contain the user's fxa uid, except for those that are recorded before the user authenticates with FxA.
+
+Events related to specific items should also have an item id in the extra field where possible.
+
+Here's an example of (something like) the swift code needed to record the event that fires when an item in the entry list is tapped:
+
+```swift
+Telemetry.default.recordEvent(
+	category: TelemetryEventCategory.action,
+	method: TelemetryEventMethod.tap,
+	object: TelemetryEventObject.entryList,
+	value: nil,
+	extras: ["fxauid" : uid, "itemid" : itemid]
+)
+```
+
+Finally, the `appName` metadata sent with each ping should always be 'Lockbox'.
+
+See here for more information on event schemas:
+
+https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/events.html#public-js-api
+
+## List of Proposed Events
+
+1. When the user starts up the app:
+	* `category`: action
+	* `method`: startup
+	* `object`: app
+	* `value`: nil
+	* `extras`: ["fxauid" : uid or nil]
+		* Note: This might be able to replace the event that currently fires in `AppDelegate.swift`
+
+2. When a user taps the fxa signin button:
+	* `category`: action
+	* `method`: tap
+	* `object`: fxaSigninButton
+	* `value`: nil
+	* `extras`: nil
+
+3. Whether a user successfully authorizes with FxA:
+	* `category`: action
+	* `method`: signin
+	* `object`: app
+	* `value`: `true` or `false`
+	* `extras`: ["fxauid" : uid or nil, "error" : nil or string]
+		* Note: If there is an authentication error, let's put it in the extra field here, if possible.
+
+4. When a user taps an item in the entry list:
+	* `category`: action
+	* `method`: tap
+	* `object`: entryList
+	* `value`: nil
+	* `extras`: ["fxauid" : uid, "itemid" : itemid]
+
+5. When a user taps one of the buttons that are shown in the menu displayed after selecting an entry:
```

Comment:
I think that menu is no longer a part of the iOS designs... can @changecourse confirm?

Reply:
Correct... we are removing all functionality around quick-access pane for now, and instead supporting standard chevron, click anywhere on the bar, to access entry details.